pkg/math_utils/utils_test.go

```diff
@@ -0,0 +1,34 @@
+package math_utils_test
+
+import (
+	. "github.com/lambdaclass/cairo-vm.go/pkg/math_utils"
+	"math/big"
+	"testing"
+)
+
+func div_mod_ok(t *testing.T) {
+	var a, b, prime, expected *big.Int
```

Comment:
this is not the proper way to initialize pointer variables because it cause a null pointer dereference error.  The way to this is like 

`a, _ := new(big.Int).SetString(..)`
thus golang creates an instance of an object where to store the data created by SetString. 
the same applies to `b` `prime` and `expected`